include/megaapi.h

```diff
@@ -14716,10 +14862,79 @@ class MegaApi
+         * and the call checkSMSVerificationCode() can be used to validate the user did receive the txt,
+         * so that really is their phone number.
+         *
+         * The frequency with which this call can be used is very limited (once or twice per
+         * 24 hour period, at the time of writing this), so it's important to get the 
+         * number right on the first try.  The result will be API_ETEMPUNAVAIL if it has
+         * been tried too frequently.
+         *
+         * Make sure to test the result of smsAllowedState() before calling this function.
          *
          * @param listener MegaRequestListener to track this request
          */
-        void catchup(MegaRequestListener *listener = NULL);
+        void sendSMSVerificationCode(const char* phoneNumber, MegaRequestListener *listener = NULL, bool reverifying_whitelisted = false);
+ 
+        /**
+         * @brief Check a verification code that the user should have received via txt
+         *
+         * This function validates that the user received the txt sent by sendSMSVerificationCode().
+         * If the user supplied the right code, the result will be API_OK.
+         *
+         * @param listener MegaRequestListener to track this request
+         */
+        void checkSMSVerificationCode(const char* verificationCode, MegaRequestListener *listener = NULL);
+
+        /**
+         * @brief Requests the contacts that are registered at MEGA (currently verified through SMS)
+         *
+         * The request will return any of the provided contacts that are registered at MEGA, i.e.,
+         * are verified through SMS (currently).
+         *
+         * contacts is a MegaStringMap from 'user detail' to the user's name. For instance:
+         * {
+         *   "+0000000010": "John Smith",
+         *   "+0000000011": "Peter Smith",
+         * }
+         *
+         * The response value is stored as a MegaStringTable with three columns:
+         * 1. entry user detail (the user detail as it was provided in the request)
+         * 2. identifier (the user's identifier)
+         * 3. user detail (the normalized user detail, e.g., +00 0000 0010)
+         *
+         * The associated request type with this request is MegaRequest::TYPE_GET_REGISTERED_CONTACTS
+         * Valid data in the MegaRequest object received in onRequestFinish when the error code
+         * is MegaError::API_OK:
+         * - MegaRequest::getMegaStringTable
+         *
+         * @param contacts The map of contacts to get registered contacts from
+         * @param listener MegaRequestListener to track this request
+         */
+        void getRegisteredContacts(const MegaStringMap* contacts, MegaRequestListener *listener = NULL);
+
+        /**
+         * @brief Requests the currently available country calling codes
+         *
+         * The response value is stored as a MegaStringListMap mapping from two-letter country code
```

Comment:
Adjust the documentation to the standard. Basically, members of the requests that are available at all callbacks, members available when the request finishes with API_OK and possible error codes with a hint for the reasons.

Reply:
will do